src/Language/PureScript/AST/Declarations.hs

```diff
@@ -390,7 +390,7 @@ data Declaration
   --
   | ValueDeclaration Ident NameKind [Binder] (Either [(Guard, Expr)] Expr)
   -- |
-  -- A binded value declaration (binder, optional guard, value)
+  -- A bound value declaration (binder, optional guard, value)
```

Comment:
Could you please expand this comment a bit? I'm not sure what it means for a declaration to be "bound", but maybe we can just say it's a declaration paired with a binder in a `let` clause.